pages/report-hazard/index.html

```diff
@@ -17,6 +17,8 @@
   <link rel="stylesheet" href="../../modules/leaflet/leaflet.css" />
   <!-- leaflet JS for maps -->
   <script src="../../modules/leaflet/leaflet.js"></script>
+  <!-- Map Controls Styles -->
+  <link rel="stylesheet" href="/assets/css/map-controls.css" />
```

Comment:
import the css file inside the style.css

Reply:
okay